test/fixtures/scenarios/emptyKibana/.kibana.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [{
```

Comment:
Can you rename this file? I know the index is `.kibana`, but it feels weird to have a hidden file with code in it here